pkg/csi/service/common/constants.go

```diff
@@ -113,6 +113,14 @@ const (
 	// on which CNS is supported.
 	MinSupportedVCenterPatch int = 3
 
+	// VSphere67u3Version is the minimum vSphere version to use Vslm APIs
+	// to support volume migration feature
+	VSphere67u3Version string = "6.7.3"
```

Comment:
I think we expect customers to use a specific patch version of vSphere 67u3

Reply:
Used buildInfo from `ServiceContent.About{}` to identify if vc version is >= update 3l ==> this is vc 6.7u3P04 && <= 7.0